src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -162,7 +166,8 @@ def show_help(self, cli_name, nouns, parser, is_group):
             AzCliHelp.update_examples(help_file)
         self._print_detailed_help(cli_name, help_file)
 
-        print(SURVEY_PROMPT)
+        from colorama import Fore, Style
```

Comment:
>from colorama import Fore, Style [](start = 8, length = 32)

suggest to add some description in PR about colorama changes, I cannot understand the difference between colorama.init(autoreset=True) and use from colorama import Fore, Style

Reply:
This is really a long store about terminal coloring. You may check the colorama doc https://pypi.org/project/colorama/ for its usage. `colorama.init(autoreset=True)` logic has been moved to Knack. 

This line is redundant. Let's remove it.